tools/monorepo-utils/src/md-docs/commands/manifest/index.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * External dependencies
+ */
+import { Command } from '@commander-js/extra-typings';
+
+/**
+ * Internal dependencies
+ */
+import { generateManifestCommand } from './create';
+
+/**
+ * Internal dependencies
+ */
+
+const program = new Command( 'md-docs' )
+	.description( 'Utilities for generating markdown doc manifests.' )
+	.addCommand( generateManifestCommand, { isDefault: true } );
+
+export default program;
```

Comment:
It seems I've broken away from the convention here with the Slack command and with this one. I can't remember my reasoning for doing that since they're functionally equivalent. I'll fix this one here and look to refactor the Slack one later.

Reply:
Resolved in https://github.com/woocommerce/woocommerce/pull/39161/commits/f2adc16b045ac88f22d3cf7245eda55b7c2b4474